docs/authoring.md

```diff
@@ -251,7 +251,7 @@ export default function({ asyncapi, params, originalAsyncAPI}) {
   );
 }
 ```
-The react component returns a `File` component as root component which the React renderer uses to figure out what what file should be generated. In our case we overwrite the default functionality of saving the file as `MyTemplate.js` but instead use the filename `asyncapi.md`. It is then specified that we should render `Some text that should render as is` within that file.
+The exported default function returns a `File` component as a root component which the [Generator](https://github.com/asyncapi/generator) uses to figure out what file should be generated. In our case we overwrite the default functionality of saving the file as `MyTemplate.js` but instead use the filename `asyncapi.md`. It is then specified that we should render `Some text that should render as is\n` within that file. Notice the `\n` character at the end, this is something that is automatically added after each children in the `File` and `Indent` component. 
```

Comment:
```md
Notice the `\n` character at the end, this is something that is automatically added after each children in the `File` and `Indent` component. 
```

It's added by each `Text` component, not after each children 😄 https://github.com/asyncapi/generator-react-sdk/blob/1e897fc2d2d6b42fb2513f067d88bb5e9b8f2c88/src/components/Text.tsx#L50